ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -777,6 +777,7 @@ open class BottomCommandingController: UIViewController, TokenizedControlInterna
         itemView.numberOfTitleLines = Constants.heroButtonMaxTitleLines
         itemView.isSelected = item.isOn
         itemView.isEnabled = item.isEnabled
+        itemView.isHidden = item.isHidden
```

Comment:
Seems like if the item is hidden, we just shouldn't be making the view. We should probably just filter out all of the hidden items in `reloadHeroCommandStack` and still show 5 items per row, rather than allow setting `isHidden` to change the number of items per row indirectly.

Reply:
Added filtering to the `heroItems` array to filter out hidden items so that this change isn't needed.